app/components/Layout/Footer.jsx

```diff
@@ -444,24 +444,9 @@ class Footer extends React.Component {
                                     />
                                 )}
                                 <span style={updateStyles}>
-                                    <Translate
-                                        content="footer.title"
-                                        wallet_name={getWalletName()}
-                                    />
-                                    {__GIT_BRANCH__ === "staging" ? (
-                                        <a
-                                            href={`https://github.com/bitshares/bitshares-ui/commit/${version.trim()}`}
-                                            className="version"
-                                            target="_blank"
-                                            rel="noopener noreferrer"
-                                        >
-                                            {version}
-                                        </a>
-                                    ) : (
                                         <span className="version">
-                                            {version}
+                                            Tatch Cloud Wallet - 0.Beta.6.ParaBellum.1
```

Comment:
I'd recommend leaving the version dynamically!

It's injected from package.json